### ISSUE TITLE
Feedback on UI

### ISSUE THREAD

**Issue by ljchang** [cosanlab/trust-game#6]
Just a few quick comments based on demo with @YounZGrace.

- [x] A lot of numbers to keep track of. UI seems a bit cluttered. I get that the idea is to minimize math in the head, but I'm having a hard time knowing where to focus. I think it would be more intuitive to have a box with bigger numbers in the center that corresponds to the slider value, instead of computing amount to me and amount to partner on each anchor. Not intuitive.
- [x] Name seems like it might be hardcoded right now and not using the name submitted by the user.
- [x]  For Trustee - It is important to see what the investment is from Player 1. On Second order belief screen, it's not obvious what the investment was. Something like "Grace invested $2.50 out of $5.00", which becomes $10.
- [x]  Multiplier should be 4 by default unless we are doing hidden multiplier version.
- [x]  It's hard to keep track of total amount of money for player 1 from player 2's perspective. 
- [x]  Summary screen is hard to follow. Here is what I recommend. Focus on Player 1's investment, "Grace invested $2.50 out of $5.00", which becomes $10."  "You returned $5.00". Grace gets $7.50 and you get $5.00".  Remove all of the ratings here. Then you can hand down Grace's expectation on the next screen.  "Grace expected you to return $5.00". On this screen you will want to make your guilt ratings. 
- [x]  Remove anger, partner's guilt, closeness, satisfaction for now to simplify to the essence.
- [x]  Let's add a counterfactual guilt rating.  How this works is you show a summary of the trial and then ask "How much guilt would you feel if you returned $XX amount of Money?"  XX will be a number that is lower than the amount actually returned.  
- [x]  In the actual experiments, we never ask guilt in between trials so as to not bias the participants to focus on guilt (or any emotion).
- [ ] #8

**Comment by ljchang** [comment]
will send more later when I have more time to play.

**Comment by wasita** [comment]
@ljchang 
can you clarify what you mean by recap? is it like 'Investor invested XX% of the endowment on average'? or something else?

**Comment by wasita** [comment]
@ejolly have updated luke's comments into a to do list so we know which tasks/comments need to be completed/addressed

**Comment by wasita** [comment]
- [x] add ITI jitter

**Comment by wasita** [comment]
- [x] blow up text etc. more clear for scanner and video shoot

**Comment by ljchang** [comment]
> @ljchang can you clarify what you mean by recap? is it like 'Investor invested XX% of the endowment on average'? or something else?

We talked about this yesterday, but basically the idea is to completely separate the ratings from the decisions. I don't think it is necessary for this demo, but is what we often do in decision-making studies to not make subject hyperfocused on emotions.

**Comment by wasita** [comment]
@ljchang yeahhh i wrote that comment asking for clarification before we met
yesterday but thanks for following up

**Comment by wasita** [comment]
- [x] linebreaks where needed in prompts 
- [x] round vals to nearest whole int
